### PR TITLE
Merging gh -pages 2020.4 branch into gh-pages branch [Merge after 2020.4 is released]

### PR DESCRIPTION
**Pull request by jainam1995**
Merging gh -pages 2020.4 branch into gh-pages branch [Merge after 2020.4 is released]

### CODE REVIEW

File: docs/api-reference.md [hunk 1]
```diff
@@ -242,6 +223,9 @@ Example:
 
 ## Deprecated API
 
+### ConnectionNormalizer and ConnectionMatcher JavaSscript files
+The JavaScript files for connection normalizer and connection matcher are deprecated as of Tableau 2020.3. This means the element  `<script file="fileName.js"/>` (which was added inside the `<connection-matcher>` and `<connection-normalizer>` element) and the `<connection-matcher>` element itself are deprecated as of 2020.3. The `<connection-normalizer>` component can be added to the connectionResolver.tdr fie as shown in the connection-normalizer section above.
```
Comment (pvanderknyff) [inline docs/api-reference.md:227]:
```suggestion
### ConnectionNormalizer and ConnectionMatcher JavaSscript files
The JavaScript files for connection normalizer and connection matcher are deprecated as of Tableau 2020.4. This means the element  `<script file="fileName.js"/>` (which was added inside the `<connection-matcher>` and `<connection-normalizer>` element) and the `<connection-matcher>` element itself are deprecated as of 2020.4. The `<connection-normalizer>` component can be added to the connectionResolver.tdr fie as shown in the connection-normalizer section above.
```

This says the normalizer and matcher are deprecated as of 2020.3, shouldn't this be 2020.4?

Comment (rosswbrown) [reply docs/api-reference.md:227]: 2020.3 is correct.  Coincides with bulk of the Gallery work and those policies.

File: docs/example.md [hunk 2]
```diff
@@ -146,46 +227,23 @@ This example is for a connector to Amazon Athena.
...
 
 For JDBC, search for <span style= "font-family: courier new">Connection URL</span> to find something like this example:
 
 ```
-JDBCProtocol Connection URL: jdbc:postgresql://postgres:5342/TestV1?user=test&password=********
+JDBCProtocol Connection URL: jdbc:postgresql://postgres:5342/TestV1
 ```
 
-## ![9]({{ site.baseurl }}/assets/pce-9.png) Tableau Dialect File
+## ![7]({{ site.baseurl }}/assets/pce-9.png)
```
Comment (rosswbrown) [inline docs/example.md:246]: I think this should still say 'Tableau Dialect File', likely just a merge issue.

Comment (jainam1995) [reply docs/example.md:246]: Oops , looks like in in the Pull request for the gh-pages 2020.4 branch the 'Tableau Dialect File' was removed mistakenly . Fixing it.

**Review by rosswbrown — APPROVED** [review]

**Review by pvanderknyff — APPROVED** [review]
LGTM

**Review by rosswbrown — APPROVED** [review]

**Pull request merged**